showkase/src/main/java/com/airbnb/android/showkase/ui/ShowkaseBrowserApp.kt

```diff
@@ -124,46 +131,49 @@ internal fun ShowkaseAppBar(
-            showkaseBrowserScreenMetadata.value.isSearchActive,
-            showkaseBrowserScreenMetadata.value.currentGroup,
-            showkaseBrowserScreenMetadata.value.currentComponentName,
-            showkaseBrowserScreenMetadata.value.currentComponentStyleName,
-            currentRoute,
-            showkaseBrowserScreenMetadata.value.searchQuery,
-            {
-                showkaseBrowserScreenMetadata.value =
-                    showkaseBrowserScreenMetadata.value.copy(searchQuery = it)
-            },
-            Modifier.fillMaxWidth(0.75f),
-            onCloseSearchFieldClick = {
-                showkaseBrowserScreenMetadata.value =
-                    showkaseBrowserScreenMetadata.value.copy(isSearchActive = false)
-            },
-            onClearSearchField = {
-                showkaseBrowserScreenMetadata.value =
-                    showkaseBrowserScreenMetadata.value.copy(searchQuery = "")
-            }
-        )
-        ShowkaseAppBarActions(
-            showkaseBrowserScreenMetadata,
-            currentRoute,
-            Modifier.fillMaxWidth(0.25f)
-        )
+        Row(
+            Modifier
+                .fillMaxWidth()
+                .padding(padding2x),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            ShowkaseAppBarTitle(
+                showkaseBrowserScreenMetadata.value.isSearchActive,
+                showkaseBrowserScreenMetadata.value.currentGroup,
+                showkaseBrowserScreenMetadata.value.currentComponentName,
+                showkaseBrowserScreenMetadata.value.currentComponentStyleName,
+                currentRoute,
+                showkaseBrowserScreenMetadata.value.searchQuery,
+                {
+                    showkaseBrowserScreenMetadata.value =
+                        showkaseBrowserScreenMetadata.value.copy(searchQuery = it)
+                },
+                Modifier.fillMaxWidth(0.75f),
+                onCloseSearchFieldClick = {
+                    showkaseBrowserScreenMetadata.value =
+                        showkaseBrowserScreenMetadata.value.copy(isSearchActive = false)
+                },
+                onClearSearchField = {
+                    showkaseBrowserScreenMetadata.value =
+                        showkaseBrowserScreenMetadata.value.copy(searchQuery = "")
+                }
+            )
+            ShowkaseAppBarActions(
+                showkaseBrowserScreenMetadata,
+                currentRoute,
+                Modifier.fillMaxWidth(0.25f)
+            )
+        }
     }
-
     /**
      * Commented out due to TopAppBar not working properly in beta-01 for this use case. Seems to be
      * related to use to Surface inside TopAppBar and a TextField. Creating my own implementation
-     * for now. Will uncomment if this issue gets fixed.
+     * for now. Wil
+     * l uncomment if this issue gets fixed.
```

Comment:
I guess it is not wanted 🤔

Reply:
🤦 
Thanks! fixed: https://github.com/airbnb/Showkase/pull/412/commits/cba224c4082a4e3d00444fb3b3762db980c89952